deploy/crds/planetscale.com_vitessclusters.yaml

```diff
@@ -1172,6 +1172,28 @@ spec:
                       type: string
                     durabilityPolicy:
                       type: string
+                    images:
+                      properties:
+                        mysqld:
+                          properties:
+                            mariadb103Compatible:
+                              type: string
+                            mariadbCompatible:
+                              type: string
```

Comment:
I think that we can remove these from all of the CRDs on main now, but for now I think we can NOT propagate them here.